src/logger/stream.ts

```diff
@@ -1,8 +1,19 @@
-import { log } from './logger';
+import { logger } from './logger';
 
 const logStream = {
   write: (message: string) => {
-    log.info(message.trim());
+    // Map Morgan log levels to Winston log levels
+    if (message.includes('error')) {
+      logger.error(message.trim());
+    } else if (message.includes('warn')) {
+      logger.warn(message.trim());
+    } else if (message.includes('info')) {
+      logger.info(message.trim());
+    } else if (message.includes('debug')) {
+      logger.debug(message.trim());
+    } else {
+      logger.info(message.trim());
+    }
```

Comment:
This is prone to misuse, I unfortunately do not have a better solution as I don't get the full scope. For example, what happens when I have "Error" in the message instead of "error"?